backend/main/models/community_users.go

```diff
@@ -30,7 +31,12 @@ var USER_TYPES = UserTypes{"member", "author", "admin"}
 
 type UserCommunity struct {
 	Community
-	Membership_type string `json:"membershipType,omitempty"`
+	Roles string
+}
+
+type UserCommunityPayload struct {
+	Community
+	Roles []string `json:"roles" validate:"required"`
```

Comment:
Can we make it return `roles:` instead of `Roles` to keep it the same with the other fields the frontend is getting as response?